docs/quickstart.rst

```diff
@@ -18,3 +18,5 @@ provides significantly more detailed information.
    </building-apps/quickstart>`
 #. :doc:`Quick start: Launching MPI applications
    </launching-apps/quickstart>`
+#. :ref:`Quick start: Customizing run-time options
```

Comment:
The other 3 quickstart guides have a specific style to them: they specifically say that they are quick starts, and skip lots of details to give the (hopefully) most important details that (hopefully) a large portion of users will need.

I don't know if simply liinking to an existing section in the doc and calling it a "quick start" fills the same requirements.

Reply:
Okay I will create a new quick start page for that

Reply:
Added a quickstart page, though I was having trouble figuring out if I should just rename the entire MCA section or split it out into different sections. Instead I thought it was probably more useful to keep it together and just added a quickstart section into the MCA section and reformatted a bit.

Reply:
I like your new split-up-the-MCA-page-into-multiple pages scheme -- it works well!